integration-tests/gradle/src/main/resources/basic-composite-build-extension-project/extensions/quarkus-example-extension/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: Quarkus Example Extension
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+metadata:
+  config:
+    - "quarkus.example.extension."
+  keywords:
+    - "logzio"
+    - "logging"
+  categories:
+    - "logging"
+#  status: "experimental"
+  description: "Quarkus example extension"
+#  built-with-quarkus-core: "2.7.1.Final"
+#  extension-dependencies:
+#    - "io.quarkus:quarkus-arc::jar"
+#    - "io.quarkus:quarkus-core::jar"
+#artifact: "com.neema:quarkus-logzio::jar:@version@"
```

Comment:
The elements commented above are supposed to be filled in by the generator, aren't they? The comments should be removed either way.